experiments/make_notebooks.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -e
+
+# Check for --skip-run flag
+skip_run=false
+for arg in "$@"
+do
+    if [ "$arg" == "--skip-run" ]; then
+        skip_run=true
+    fi
+done
+
+required_commands=("jupytext" "papermill" "python")
+
+# Loop over each required command
+for command in "${required_commands[@]}"; do
+    # Check if the command is installed
+    if ! command -v $command &> /dev/null
+    then
+        echo "$command could not be found"
+        echo "Please install it using 'pip install $command'"
+        exit
+    fi
+done
+
+# Define the file paths
+declare -A file_paths
+file_paths=(
```

Comment:
We should do this with a `Makefile`, it's much cleaner and can be run automatically in parallel. It should remain TODO for another PR though.